test/MathOptInterface/MOI_wrapper.jl

```diff
@@ -120,3 +120,25 @@ end
         end
     end
 end
+
+@testset "Conflict refiner" begin
+    @testset "Variable bounds" begin
+        # Same test as ../C_API/iis.jl, but ported to MOI. 
+        model = CPLEX.Optimizer()
+        x = MOI.add_variable(model)
+        b = MOI.add_constraint(model, MOI.SingleVariable(x), MOI.GreaterThan(0.0))
+        c1 = MOI.add_constraint(model, MOI.SingleVariable(x), MOI.GreaterThan(2.0))
+        c2 = MOI.add_constraint(model, MOI.SingleVariable(x), MOI.LessThan(1.0))
```

Comment:
This isn't the same test. Use `MOI.ScalarAffineFunction(MOI.ScalarAffineTerm.([1.0], [x]), 0.0), MOI.LessThan(1.0))` to test scalaraffine constraints.